__tests__/api.test.js

```diff
@@ -328,5 +328,43 @@ describe('GET /api/reviews/:review_id/comments', () => {
             })
         })
     })
-    test.todo('');
+    test('200: responds with an object containing an empty array when a valid review id that has no comments is requested', () => {
+        const testReviewID = 6;
+        return request(app)
+        .get(`/api/reviews/${testReviewID}/comments`)
+        .expect(200)
+        .then(({body}) => {
+            expect(body.hasOwnProperty('comments')).toBe(true);
+            expect(Array.isArray(body.comments)).toBe(true);
+            expect(body.comments.length).toBe(0);
+        })
+    });
+    test('404: responds with "Review not found" when a valid review id that does not exist is requested', () => {
+        const testReviewID = 300;
+        return request(app)
+        .get(`/api/reviews/${testReviewID}/comments`)
+        .expect(404)
+        .then(({body}) => {
+            expect(body).toEqual({msg: "Review not found"})
+        })
+    });
+    test('400: responds with "Bad Request" when requesting the comments for an invalid review id', () => {
+        const testReviewID = 'not_an_id';
+        return request(app)
+        .get(`/api/reviews/${testReviewID}/comments`)
+        .expect(400)
+        .then(({body}) => {
+            expect(body).toEqual({msg: "Bad Request"})
+        })
+    });
+    test('404: responds with "Not found" if invalid endpoint is used with a valid review_id', () => {
+        const testReviewID = 3;
+        return request(app)
+        .get(`/api/reviews/${testReviewID}/invalid_endpoint`)
+        .expect(404)
+        .then(({body}) => {
+            expect(body).toEqual({msg: 'Not found'});
+        })
+    });
```

Comment:
I'd suggest you dont need this as it would be caught by the not a route error handler in the app, anyway